taxi/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
```

Comment:
Add styles to the `styles.css` and update a homepage screenshot to demonstrate that static files are loaded correctly.